src/datumbazo/pagination.clj

```diff
@@ -0,0 +1,69 @@
+(ns datumbazo.pagination
+  (:require [no.en.core :refer [parse-integer]]
+            [sqlingvo.core :as sql]))
+
+(def ^:dynamic *page*
+  "The current page number."
+  nil)
+
+(def ^:dynamic *per-page*
+  "The default number of pages."
+  25)
```

Comment:
Default values should be defined at the .graphql schema level this is a feature of graphql to also standardize those things.